closed/make/copy/Copy-java.base.gmk

```diff
@@ -35,8 +35,9 @@ TARGETS += \
 	$(INCLUDE_TARGET_DIR)/ibmjvmti.h \
 	#
 
-$(INCLUDE_TARGET_DIR)/jvmti.h : $(TOPDIR)/openj9/runtime/include/jvmti.h
-	$(call install-file)
+$(INCLUDE_TARGET_DIR)/jvmti.h : $(TOPDIR)/openj9/runtime/include/jvmti.h.m4
+	$(MKDIR) -p $(OUTPUTDIR)/vm/include
+	m4 -D JAVA_SPEC_VERSION=$(VERSION_FEATURE) $< | tee -a $@ $(OUTPUTDIR)/vm/include/jvmti.h > /dev/null
```

Comment:
If the file in `$(OUTPUTDIR)/vm` is needed in addition to the target of this rule, split the rule into two to produce both, but each rule should normally create the target it claims to produce. Use `$(@D)` to refer to the directory containing the target file rather than repeating paths.

Please remove the use of `tee` - particularly the use of the `-a` option which will append to an existing file rather than create the desired content.

Reply:
Do you mean that split `$(INCLUDE_TARGET_DIR)/jvmti.h` and `$(OUTPUTDIR)/vm/include/jvmti.h`? If so, I have split them into two rules and test building successfully in my local machine.